src/main/resources/test_files/runtime-spark/Pushdown/KeysDF1.jq

```diff
@@ -1,2 +1,2 @@
-(:JIQS: ShouldRun; Output="(choices, country, date, guess, sample, target)" :)
+(:JIQS: ShouldRun; Output="(country, choices, guess, target, date, sample)" :)
 keys(structured-json-file("./src/main/resources/queries/conf-ex.json"))
```

Comment:
This test is executed locally on the master which I believe is not correct. It is now executed with RDD API on the keys iterator which gives the new results.